HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/ExtractionResult.swift

```diff
@@ -25,6 +25,7 @@ public enum ExtractionType: String {
     case paymentRecipient = "payment_recipient"
     case iban = "iban"
     case paymentPurpose = "payment_purpose"
+    case doctorName = "medical_service_provider"
```

Comment:
[Clarification]: Do we use it in SDK? As I understood no, means we have here a convenience enum, is it correct?

Reply:
Yes, it's a helper used for clients now.  Because we still haven't migrated to BaseSDKs. We use this in GiniHealthSDKExample app as an example

Reply:
https://health-api.gini.net/documentation/v4/#extractions
could you please add to the helper 2 more cases for `bic` and `invoice_date` then we will have all keys in 1 place